README.md

```diff
@@ -276,6 +276,36 @@ const opts = {
 get(opts, function (err, res) {})
 ```
 
+### Throttle requests
+
+You can use [limiter](https://github.com/jhurliman/node-rate-limiter) to throttle requests. This is useful when calling an API that is rate limited.
+
+```js
+const simpleGet = require('simple-get');
+const RateLimiter = require('limiter').RateLimiter;
+const limiter = new RateLimiter(1, 'second'); // one request per second
+
+// covers both get() and get.concat()
+const get = (opts, cb) => limiter.removeTokens(1, () => simpleGet(opts, cb));
+get.concat = (opts, cb) => limiter.removeTokens(1, () => simpleGet.concat(opts, cb));
+
+const opts = {
+  url: 'http://example.com'
+};
+
+get(opts, processResult);
+get.concat(opts, processResult);
```

Comment:
Can you just use one of the methods here? I think that would make the example clearer. It's fine to rate limit both methods, but I would just use one them, say `get.concat`.